test/src/cfile/test_cfile.cpp

```diff
@@ -0,0 +1,29 @@
+
+#include <gtest/gtest.h>
+
+#include <cfile/cfile.h>
+
+#include "util/FSTestFixture.h"
+
+class CFileTest : public test::FSTestFixture {
+ public:
+	CFileTest() : test::FSTestFixture(0) {
+		pushModDir("cfile");
```

Comment:
`pushModDir` simply adds another `-mod` parameter so it seems like the subdirectory implementation doesn't handle mods correctly.